src/test/store/fixtures/profiles.js

```diff
@@ -34,13 +37,23 @@ export function getProfileWithMarkers(markers: TestDefinedMarkers): Profile {
   };
 
   markers.forEach(([name, time, data]) => {
+    if (data && !data.type) {
+      data = {
+        type: 'DummyForTests',
```

Comment:
I added this dummy type -- I'm not entirely satisfied, I'd prefer that our tests use real-life markers, but this is a lot of noise for this patch.